app/controllers/state_file/questions/az_extension_payments_controller.rb

```diff
@@ -1,13 +1,17 @@
 module StateFile
   module Questions
     class AzExtensionPaymentsController < QuestionsController
-      def self.show?(intake)
-        Flipper.enabled?(:extension_period) && after_april_16_arizona_time?
+      # Class method called externally (e.g., in a flow)
+      def self.show?(intake, current_time: Time.current)
+        Flipper.enabled?(:extension_period) && after_filing_deadline?(intake, current_time)
```

Comment:
Options to use `app_time`:
1. Custom logic for this method only since it's the only time we call `app_time` from a class method (which doesn't have access to `app_time` method on Application Controller:
 -  if rails.env is not prod, then `SessionToggle.new(session, 'app_time').value`
2. Find a place outside of `ApplicationController` to house the `app_time` logic & point to that method from ApplicationController, use the method directly here

Reply:
@mpidcock do you have a good idea for where we can move the `app_time` method so we can access it from the class method?